tests/test_batch.py

```diff
@@ -794,7 +794,7 @@ def test_drop_fk(self):
         new_table = self._assert_impl(
             impl,
             colnames=["id", "email", "user_id"],
-            ddl_not_contains="CONSTRANT fk1",
+            ddl_not_contains="CONSTRAINT fk1",
```

Comment:
afaict this means that the test didn't test what it thought it was testing

Reply:
I'll look into it